contrib/spouts/kafka/java/heron-kafka-spout-parent/heron-kafka-spout/src/main/java/org/apache/heron/spouts/kafka/KafkaSpout.java

```diff
@@ -330,11 +355,8 @@ private void manualCommit(TopicPartition topicPartition, NavigableMap<Long, Long
   private Iterable<ConsumerRecord<K, V>> poll() {
     ConsumerRecords<K, V> records = consumer.poll(Duration.ofMillis(200));
     if (!records.isEmpty()) {
-      //since the Kafka Consumer metrics are built gradually based on the partitions it consumes,
-      //we need to periodically check whether there's any new metrics to register after
-      //each polling.
       if (System.currentTimeMillis() - previousKafkaMetricsUpdatedTimestamp
-          > metricsIntervalInSecs) {
+          > metricsIntervalInSecs * 1000) {
```

Comment:
why "* 1000"?

Reply:
because `metricsIntervalInSecs` has the unit of second, while the duration is in millisecond

Reply:
Ah. My bad. I dont know what I was thinking.